thrift/thrift.bzl

```diff
@@ -135,8 +148,7 @@ thrift_library = rule(
       "absolute_prefixes": attr.string_list(),
       # This is a list of JARs which only contain Thrift files
       "external_jars": attr.label_list(),
-      "_jar": attr.label(executable=True, cfg="host", default=Label("@bazel_tools//tools/jdk:jar"), allow_files=True),
-      "_jdk": attr.label(default=Label("//tools/defaults:jdk"), allow_files=True),
+      "_zipper": attr.label(executable=True, cfg="host", default=Label("@bazel_tools//tools/zip:zipper"), allow_files=True)
```

Comment:
TIL this exsited! This kind of thing needs better publicity. I've cargo culted around the restamped archive code in like every archive rule I've written

Reply:
right?